python_modules/dagster-test/dagster_test_tests/test_graph_job_op_toys.py

```diff
@@ -6,83 +6,99 @@
-    IntentionalRandomFailure,
-    longitudinal_pipeline,
+from dagster_test.graph_job_op_toys.asset_lineage import asset_lineage_job
+from dagster_test.graph_job_op_toys.branches import branch_failed_job, branch_job
+from dagster_test.graph_job_op_toys.composition import composition_job
+from dagster_test.graph_job_op_toys.dynamic import dynamic_job
+from dagster_test.graph_job_op_toys.error_monster import (
+    define_errorable_resource,
+    error_monster,
+    errorable_io_manager,
 )
-from dagster_test.graph_job_op_toys.many_events import many_events
-from dagster_test.graph_job_op_toys.pyspark_assets.pyspark_assets_pipeline import (
-    pyspark_assets_pipeline,
+from dagster_test.graph_job_op_toys.hammer import hammer_default_executor_job
+from dagster_test.graph_job_op_toys.log_spew import log_spew_job
+from dagster_test.graph_job_op_toys.longitudinal import IntentionalRandomFailure, longitudinal_job
+from dagster_test.graph_job_op_toys.many_events import many_events_job
+from dagster_test.graph_job_op_toys.pyspark_assets.pyspark_assets_job import (
+    dir_resources,
+    pyspark_assets_graph,
+    pyspark_assets_job,
 )
 from dagster_test.graph_job_op_toys.repo import toys_repository
-from dagster_test.graph_job_op_toys.resources import resource_pipeline
-from dagster_test.graph_job_op_toys.retries import retry_pipeline
+from dagster_test.graph_job_op_toys.resources import lots_of_resources, resource_graph, resource_job
+from dagster_test.graph_job_op_toys.retries import retry_job
 from dagster_test.graph_job_op_toys.schedules import longitudinal_schedule
-from dagster_test.graph_job_op_toys.sleepy import sleepy_pipeline
+from dagster_test.graph_job_op_toys.sleepy import sleepy_job
 
 
 def test_repo():
     assert toys_repository
 
 
-def test_dynamic_pipeline():
-    assert execute_pipeline(dynamic_pipeline).success
+def test_dynamic_job():
+    assert dynamic_job.execute_in_process().success
 
 
-def test_longitudinal_pipeline():
+def test_longitudinal_job():
     partition_set = longitudinal_schedule().get_partition_set()
     try:
-        result = execute_pipeline(
-            longitudinal_pipeline,
+        result = longitudinal_job.execute_in_process(
             run_config=partition_set.run_config_for_partition(partition_set.get_partitions()[0]),
         )
         assert result.success
     except IntentionalRandomFailure:
         pass
 
 
-def test_many_events_pipeline():
-    assert execute_pipeline(many_events).success
+def test_many_events_job():
+    assert many_events_job.execute_in_process().success
 
 
 def get_sleepy():
-    return sleepy_pipeline
+    return sleepy_job
 
 
-def test_sleepy_pipeline():
+def test_sleepy_job():
     with instance_for_test() as instance:
         assert execute_pipeline(reconstructable(get_sleepy), instance=instance).success
 
 
-def test_spew_pipeline():
-    assert execute_pipeline(log_spew).success
+def test_branch_job():
+    with instance_for_test() as instance:
```

Comment:
[1] shouldn't need instance_for_test() when using `execute_in_process()`. `instance_for_test` is creating a persistent instance, which is necessary when using one of the out-of-process executors. Since execute_in_process is strictly in memory, I don't think this should be necessary.

Reply:
Makes sense, removed